jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Jar.java

```diff
@@ -80,37 +80,37 @@
 
   @CommandLine.Option(
       names = "--jvm-flags",
-      paramLabel = "<jvm-flags>",
+      paramLabel = "<jvm-flag>",
```

Comment:
Isn't this only for textual message in the help text? Previously, I thought showing
```
--jvm-flags <jvm-flags> JVM arguments, ...
```
made sense, because you'll provide a list of flags. Why should it be in a singular form?

Reply:
I initially thought so too. But it shows up as  `--jvm-flags=<jvm-flags>[,<jvm-flags>...]  JVM arguments, example: --jvm-flags=-Dmy.property=value,-Xshare:off` which seems a little confusing. For additional-tags, we do  `--additional-tags=<tag>[,<tag>...] Additional tags for target image`